### PR TITLE
Fix Ruby formatter

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/seamapi/docs/issues/401

### CODE REVIEW

**Review by DebbieAtSeam — APPROVED** [review]

**Comment by razor-x** [comment]
@andrii-balitskyi Are we no longer using standard? If so, we need to removed or update the references:

- Remove or update the requirments: https://github.com/seamapi/docs?tab=readme-ov-file#optional-requirements
- Make sure the devcontainer works with the updated formatter: https://github.com/seamapi/docs/blob/main/.devcontainer/Dockerfile#L8

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -48,8 +48,8 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: '^1.23.2'
-      - name: Install standardrb
-        run: gem install standard -v 1.31.0
+      - name: Install syntax_tree
+        run: gem install syntax_tree
```
Comment (razor-x) [inline .github/workflows/generate.yml:52]:
```suggestion
        run: gem install syntax_tree -v 6.2.0
```

**Review by razor-x — APPROVED** [review]

File: docs/api/acs/access_groups/get.md [hunk 2]
```diff
@@ -69,7 +69,18 @@ seam.acs.access_groups.get(acs_access_group_id: "44444444-4444-4444-4444-4444444
 #### Response
 
 ```ruby
-{"acs_access_group_id" => "44444444-4444-4444-4444-444444444444", "name" => "Lobby Access", "display_name" => "Lobby Access", "access_group_type_display_name" => "PTI access level", "access_group_type" => "pti_access_level", "external_type" => "pti_access_level", "external_type_display_name" => "PTI access level", "acs_system_id" => "11111111-1111-1111-1111-111111111111", "workspace_id" => "00000000-0000-0000-0000-000000000000", "created_at" => "2023-11-30T06:27:15.437Z"}
+{
+  "acs_access_group_id" => "44444444-4444-4444-4444-444444444444",
```
[the remark this answers is not in the document]
Comment (andrii-balitskyi) [reply docs/api/acs/access_groups/get.md:73]: I think it's ok. @razor-x and I agreed on using the hash arrow syntax for Ruby responses, similar to JSON.

**Comment by andrii-balitskyi** [comment]
@razor-x Why does devcontainer need to install ruby formatter? I don't see it installing prettier, for example. 
How formatters are used in the devcontainer?

**Comment by razor-x** [comment]
> @razor-x Why does devcontainer need to install ruby formatter? I don't see it installing prettier, for example. 
> How formatters are used in the devcontainer?

The devcontainer installs prettier as part of `npm i` because it's a dependency in package.json

**Pull request merged**